examples/app-pg/main.py

```diff
@@ -55,15 +57,15 @@ def setup_table(cur, data):
     cur.commit()
 
 
-def fetch_data(cur):
+def fetch_data(cur: Cursor) -> List:
     ''' Fetch all data from the table. '''
     print('List of data.')
     cur.execute('SELECT * FROM users')
 
     return cur.fetchall()
 
 
-def display_data(rows):
+def display_data(rows: List):
```

Comment:
Is there a way to hint what the `List` contains? This function expects `row` to include `id`, `name` and `city`. Can we show this information somehow?

Reply:
Like this?
```python 
def display_data(rows: List[Tuple[int, str, str]])
```

Reply:
Yes. This is better.